torch/quantization/fx/quantization_patterns.py

```diff
@@ -767,14 +767,19 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
 @register_quant_pattern(torch.nn.InstanceNorm3d)
 @register_quant_pattern(torch.nn.LayerNorm)
 @register_quant_pattern(torch.nn.SiLU)
+# @register_quant_pattern(torch.nn.GELU)
+# @register_quant_pattern(torch.nn.Softmax)
```

Comment:
can we add a comment that this is commented out because we only support reference patterns for these and not a real kernel?

Reply:
yeah, done